docs/user_guide/sd_fingertips.rst

```diff
@@ -1,9 +1,115 @@
 Fingertips
 ============
 
+STF Sensor
+----------
+
+.. important:: The STF sensors are prototype devices provided "as is".
+   A few caveats should be considered when operating these sensors:
+
+   • When powered up, the electronic components of the sensors warm up until they reach a stable temperature. During this period, the sensors' magnetic readings are prone to changes.
+   • Upon powering up the hand, allow 30 minutes for the sensors' internal temperatures to stabilise.
+   • Changes to the room temperature and between the room temperature and the temperature of the interacted objects may also incur changes in the magnetic readings until the internal temperature of the sensors stabilises.
+   • The sensors are protected against cross-interference with each other. However, the sensors' magnetic readings are prone to interference from strong external magnetic fields, including interference from the magnetic sensors installed in Shadow hands' joints.
+
+The Shadow Tactile Fingertip (STF) sensors are magnetic-based tactile sensors fitted in the
+standard fingertips or (slightly bigger) thumb shapes.
+
```

Comment:
Not sure about this but should we mention the dropped frames that result from these sensors??

Reply:
Ah yes, good point